inhouse/command_handlers/match.py

```diff
@@ -209,13 +214,16 @@ def create_active_db_entry(self):
         Helper to create an active_matches db entry for this match
         """
         self.create_missing_players()
-
         player_ids_str = ""
-        for key in roles:
-            player_ids_str += f"'{str(self.blue_team[key].id)}','{str(self.red_team[key].id)}',"
+        if not self.is_test_match:
+            for key in roles:
+                player_ids_str += f"'{str(self.blue_team[key].id)}','{str(self.red_team[key].id)}',"
         
         # strip last comma and insert
-        cmd = f"INSERT INTO active_matches({all_roles_db_key}) VALUES ({player_ids_str.strip(',')}) RETURNING active_id"
+        if self.is_test_match:
+            cmd = f"INSERT INTO active_matches(top1) VALUES (NULL) RETURNING active_id"
+        else:
+            cmd = f"INSERT INTO active_matches({all_roles_db_key}) VALUES ({player_ids_str.strip(',')}) RETURNING active_id"
```

Comment:
I think this can be combined and cleaned up a little like
```python
if self.is_test_match:
    # Do test match cmd
else:
    # do the key in roles loop and normal cmd
```

Reply:
Updated